src/core/constraints.jl

```diff
@@ -52,18 +52,6 @@ const NODAL_BALANCE_REACTIVE = "nodal_balance_reactive"
 
 abstract type ConstraintType end
 
-function make_constraint_name(::Type{T}) where {T <: ConstraintType}
```

Comment:
Dheepak convinced me that I shouldn't use this pattern any more.  I still think it is a good pattern that makes up for a useful feature that Julia lacks.  However, we should use `Julian` patterns.